src/CompanyService.Models.Broker/Responses/IGetUsersDataResponse.cs

```diff
@@ -0,0 +1,18 @@
+﻿using LT.DigitalOffice.Broker.Models;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.Broker.Responses
+{
+    public interface IGetUsersDataResponse
+    {
+        List<UserData> UsersData { get; set; }
+
```

Comment:
I think, no need to use "set" because of CreateObj method below.

Reply:
Fixed.